Streetcode/Streetcode.BLL/Services/Instagram/InstagramService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Text.Json;
+using Microsoft.Extensions.Options;
+using Streetcode.BLL.Interfaces.Instagram;
+using Streetcode.DAL.Entities.Instagram;
+
+namespace Streetcode.BLL.Services.Instagram
+{
+    public class InstagramService : IInstagramService
+    {
+        private readonly HttpClient _httpClient;
+        private readonly InstagramEnvirovmentVariables _envirovment;
+        private readonly string _userId;
+        private readonly string _accessToken;
+        private int postLimit = 10;
+
+        public InstagramService(IOptions<InstagramEnvirovmentVariables> instagramEnvirovment)
+        {
+            _httpClient = new HttpClient();
+            _envirovment = instagramEnvirovment.Value;
+            _userId = _envirovment.InstagramID;
+            _accessToken = _envirovment.InstagramToken;
+        }
+
+        public async Task<IEnumerable<InstagramPost>> GetPostsAsync()
+        {
+            var postLimit = 10;
```

Comment:
why do you need two postLimit variables? Also consider defining it like a constant.